ludwig/features/sequence_feature.py

```diff
@@ -226,7 +226,7 @@ def add_feature_data(
 
 
 class SequenceInputFeature(SequenceFeatureMixin, InputFeature):
-    encoder = "embed"
+    encoder = "parallel_cnn"
```

Comment:
I felt like this was a typo so added this change into this PR. Official [ludwig documentation](https://ludwig-ai.github.io/ludwig-docs/0.5/configuration/features/sequence_features/#sequence-input-features-and-encoders) says that the default encoder for sequence features is a parallel_cnn